packages/cli/src/decorators/debounce.ts

```diff
@@ -0,0 +1,22 @@
+import debounce from 'lodash/debounce';
+
+export const Debounce =
```

Comment:
Oh nice, love this decrator 💘 Could we add documentation for it with a usage example?

Reply:
Please see [16afd1e](https://github.com/n8n-io/n8n/pull/11007/commits/16afd1e33389b13b6e4118f6e48185c681db7dd1)